### PR TITLE
update app after server update

### PR DESCRIPTION
**Pull request by elzbthbrst**
update deleting chat from folder, function for creating kebab, add error-modal for error user fetch, update creating/updating folder

### CODE REVIEW

File: src/components/chatWindow/WindowChat.jsx [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
 
-
-    
     // set user first time
     useEffect(() => {
         if (user?.id) {
-            dispatch(setFetch())
-            dispatch(fetchUser(user.id));
+            dispatch(singleUserFetch(user.id));
+            console.log('fetch');
```
Comment (vlaskiri) [inline src/components/chatWindow/WindowChat.jsx:27]: delete console.log

File: src/components/chatWindow/navigation/add-folder/AddFolderModal.jsx [hunk 2]
```diff
@@ -53,15 +53,16 @@ export default function AddFolderModal() {
 
     //form submit
     function onFormSubmit(data) {
-        if (!data.publicChatIds) {
-            data.publicChatIds = []
+        console.log(data);
```
Comment (vlaskiri) [inline src/components/chatWindow/navigation/add-folder/AddFolderModal.jsx:56]: delete console.log

**Review by vlaskiri — DISMISSED** [review]

File: src/components/chatWindow/navigation/add-folder/AddFolderModal.jsx (hunk 2 above)
Comment (elzbthbrst) [reply src/components/chatWindow/navigation/add-folder/AddFolderModal.jsx]: done

File: src/components/chatWindow/WindowChat.jsx (hunk 1 above)
Comment (elzbthbrst) [reply src/components/chatWindow/WindowChat.jsx]: done

**Review by vlaskiri — APPROVED** [review]

**Pull request merged**